src/app/pages/send/send-crypto-asset-form/components/recipient-accounts-drawer/recipient-accounts-drawer.tsx

```diff
@@ -23,9 +23,9 @@ export const RecipientAccountsDrawer = memo(() => {
 
   return (
     <BaseDrawer title="My accounts" isShowing onClose={onGoBack}>
-      <Box mb="loose" mx={['base-loose', 'extra-loose']}>
+      <Box mb="space.05" mx={['space.04', 'space.06']}>
```

Comment:
There is no size for `base-loose` in the new system so I have set it to `base` as I believe we agreed previously